nrf-softdevice/Cargo.toml

```
@@ -32,7 +32,7 @@ ble-sec = []
 
 critical-section-impl = ["critical-section/restore-state-bool"]
 
-nightly = []
+nightly = ["embedded-storage-async"]
```

```suggestion
nightly = ["dep:embedded-storage-async"]
```

optional deps implicitly create a Cargo feature, unless you mention them with `dep:`. This'll disallow people from doing `nrf-softdevice = { .., features = ["embedded-storage-async"] }`, which would compile but not work.

---

done